frontend/packages/local-storage-operator-plugin/src/utils/index.ts

```diff
@@ -44,3 +47,10 @@ export const getHostNames = (nodes: string[], hostNamesMap: HostNamesMap) => {
     return [...acc, hostNamesMap[node]];
   }, []);
 };
+
+export const getNodesByHostNameLabel = (nodes: NodeKind[]): Set<string> =>
+  nodes.reduce(
+    (names: Set<string>, node: NodeKind) =>
+      names.add(node.metadata?.labels?.['kubernetes.io/hostname']),
+    new Set<string>(),
+  );
```

Comment:
```suggestion
 [...new Set(nodes.map(getHostName))]
```
Set is useful for avoiding duplicates here but we don't need to expose it as a Set. Exposing as array is better.

Reply:
Re-thinking, that we dont need a Set at all.

Reply:
reason being hostnames are unique for every node.